webpack.config.js

```diff
@@ -24,6 +24,15 @@ module.exports = {
     filename: 'bundle.js'
   },
 
+  externals: [
```

Comment:
This change prevents react from being duplicated; i saw multiple copies of react in the final bundle. it could be because i'm using npm 2.x and webpack dedupe isn't being used.